[2024-07-30T10:26:45.558+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T10:26:45.612+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-05T00:00:00+00:00 [queued]>
[2024-07-30T10:26:45.630+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-05T00:00:00+00:00 [queued]>
[2024-07-30T10:26:45.631+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T10:26:45.751+0000] {taskinstance.py:2330} INFO - Executing <Task(DataQualityOperator): Run_data_quality_checks> on 2018-11-05 00:00:00+00:00
[2024-07-30T10:26:45.760+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=982) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-30T10:26:45.762+0000] {standard_task_runner.py:63} INFO - Started process 984 to run task
[2024-07-30T10:26:45.760+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipelinev5', 'Run_data_quality_checks', 'scheduled__2018-11-05T00:00:00+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpgyo5a863']
[2024-07-30T10:26:45.763+0000] {standard_task_runner.py:91} INFO - Job 111: Subtask Run_data_quality_checks
[2024-07-30T10:26:45.954+0000] {task_command.py:426} INFO - Running <TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-05T00:00:00+00:00 [running]> on host c3b60429ae20
[2024-07-30T10:26:46.354+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipelinev5' AIRFLOW_CTX_TASK_ID='Run_data_quality_checks' AIRFLOW_CTX_EXECUTION_DATE='2018-11-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2018-11-05T00:00:00+00:00'
[2024-07-30T10:26:46.356+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T10:26:46.418+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songplays table
[2024-07-30T10:26:46.469+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:46.506+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songplays, parameters: None
[2024-07-30T10:26:46.514+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:46.518+0000] {data_quality_operator.py:30} INFO - Data quality on table songplays check passed with 920 records
[2024-07-30T10:26:46.520+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for users table
[2024-07-30T10:26:46.547+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:46.582+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM users, parameters: None
[2024-07-30T10:26:47.457+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:47.461+0000] {data_quality_operator.py:30} INFO - Data quality on table users check passed with 63 records
[2024-07-30T10:26:47.463+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songs table
[2024-07-30T10:26:47.485+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:47.503+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songs, parameters: None
[2024-07-30T10:26:47.513+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:47.516+0000] {data_quality_operator.py:30} INFO - Data quality on table songs check passed with 71 records
[2024-07-30T10:26:47.518+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for artists table
[2024-07-30T10:26:47.535+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:47.557+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM artists, parameters: None
[2024-07-30T10:26:47.561+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:47.565+0000] {data_quality_operator.py:30} INFO - Data quality on table artists check passed with 69 records
[2024-07-30T10:26:47.566+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for times table
[2024-07-30T10:26:47.579+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:47.603+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM times, parameters: None
[2024-07-30T10:26:47.616+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:47.619+0000] {data_quality_operator.py:30} INFO - Data quality on table times check passed with 920 records
[2024-07-30T10:26:47.620+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T10:26:47.653+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipelinev5, task_id=Run_data_quality_checks, run_id=scheduled__2018-11-05T00:00:00+00:00, execution_date=20181105T000000, start_date=20240730T102645, end_date=20240730T102647
[2024-07-30T10:26:47.912+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-30T10:26:47.941+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-30T10:26:47.989+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
